docs: mention `--no-deps` in a few places (#35252)
microsoft/playwright: docs/src/test-global-setup-teardown-js.md

});
```

### Test filtering

> If `--grep/--grep-invert` or `--shard` [option](./test-cli.md#reference) is used, test file name filter is specified in [command line](./test-cli.md) or [test.only()](./api/class-test.md#test-only) is used, it will only apply to the tests from the deepest projects in the project dependency chain. In other words, if a matching test belongs to a project that has project dependencies, Playwright will run all the tests from the project dependencies ignoring the filters.

Reviewer: ```suggestion
All test filtering options (such as `--grep`/`--grep-invert`, `--shard`, filtering directly by test name in the CLI invocation, or using [`test.only()`](./api/class-test.md#test-only)) directly select the primary tests to be run. If those tests are referenced by a project with project dependencies, _all_ tests from those dependencies will run.
```

> You can pass `--no-deps` command-line option to ignore dependencies and teardowns specified in the config. This way, test runner will not automatically run dependencies and teardowns needed for filtered tests.

Reviewer: ```suggestion
You can pass `--no-deps` command line option to ignore all dependencies and teardowns. Only your directly selected projects will run.
```

"command line" is referred throughout the docs without the hyphen.
